### PR TITLE
Apply python formatters,  use utf-8 and remove built-ins

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

**Comment by ethanwhite** [comment]
Looks like the builds are failing on the regression tests. I tried restarting them and that didn't change anything. Not sure if this is the issue we've been having with CI or something to fix/update in the PR.

File: test/test_retriever.py [hunk 1]
```diff
@@ -99,7 +96,7 @@ def test_auto_get_datatypes():
     The function adds 100 to the auto detected length of column
     """
     test_engine.auto_get_datatypes(None,
-                                   [["�", 'bb', 'L�ve']],
+                                   [["ö", 'bb',  'Löve']],
```
Comment (henrykironde) [inline test/test_retriever.py]: TODO: Remove extra space during clean up

File: retriever/lib/templates.py [hunk 2]
```diff
@@ -5,22 +5,34 @@
 from __future__ import print_function
 
 from retriever.engines import choose_engine
-from retriever.lib.models import *
+# from retriever.lib.models import *
```
Comment (henrykironde) [inline retriever/lib/templates.py]: TODO: remove

**Comment by henrykironde** [comment]
@ethanwhite sorry this took long but some back compatibility issues were causing the tests to mutually fail on OS nt or Unix.

**Comment by ethanwhite** [comment]
This all looks great. I suggest that we rebase the branch and re-write the relevant commit messages to add scls19fr in a Co-authored-by line (https://stackoverflow.com/questions/7442112/how-to-attribute-a-single-commit-to-multiple-developers).

If that's difficult for some reason then I think adding the Co-authored-by line to the PR message would be another way to provide some direct credit. Looks like you started that but didn't finish it.

There are also a couple of TODO's that you've left for yourself that aren't implemented yet.

Other than that this is ready to merge. Ping me once you're ready for it to go in and I'll push the button.

**Comment by ethanwhite** [comment]
Great work @henrykironde and a big thanks to @scls19fr and @Jaffrey98 for their important contributions to the Python formatters!

**Pull request merged**